lib/opentok.js

```diff
@@ -477,6 +478,168 @@ OpenTok = function (apiKey, apiSecret, env) {
     );
   };
 
+/**
+   * Retrieves a List of {@link Render} objects, representing any renders in the starting,
+   * started, stopped or failed state, for your API key.
+   *
+   * @param options {Object} An options parameter with three properties:
+   *
+   * <ul>
+   *
+   *   <li>
+   *     <code>count</code> &mdash; The maximum number of renders to return. The default number of
+   *     renders returned is 50 (or fewer, if there are fewer than 50 renders). The method returns
+   *     a maximum of 1000 renders.
+   *   </li>
+   *
+   *   <li>
+   *     <code>offset</code> &mdash; The offset for the first render to list (starting with the
+   *     first render recorded as offset 0). 1 is the offset of the render that started prior
+   *     to the most recent render. This property is optional; the default is 0.
+   *   </li>
+   *
+   *   <li>
+   *     <code>sessionId</code> &mdash; Specify the ID of a session in order to retrieve renders
+   *     specifically for that session. This property is optional. When no session ID is specified,
+   *     then the method will return renders from any session created with your API key.
+   *   </li>
+   *
+   * </ul>
+   *
+   * <p>If you don't pass in an <code>options</code> argument,
+   * the method returns up to 1000 renders
+   * starting with the first render recorded.
+   *
+   * @param callback {Function} The function to call upon completing the operation. Two arguments
+   * are passed to the function:
+   *
+   * <ul>
+   *
+   *   <li>
+   *      <code>error</code> &mdash; An error object (if the call to the method fails).
+   *   </li>
+   *
+   *   <li>
+   *       <code>renders</code> &mdash; An array of {@link Render} objects.
+   *   </li>
+   *
+   * </ul>
+   *
+   * @method #listRenders
+   * @memberof OpenTok
+   */
+  this.listRenders = render.listRenders.bind(null, apiConfig);
+
+  /**
+   * Gets an {@link Render} object for the given render ID.
+   *
+   * @param renderId {String} The render ID.
+   *
+   * @param callback {Function} The function to call upon completing the operation. Two arguments
+   * are passed to the function:
+   * <ul>
+   *   <li><code>error</code> &mdash; An error object (if the call to the method fails). </li>
+   *   <li><code>render</code> &mdash; The {@link Render} object.</li>
+   * </ul>
+   *
+   * @method #getArchive
+   * @memberof OpenTok
+   */
+  this.getRender = render.getRender.bind(null, apiConfig);
+
+  /**
+  * Starts rendering an OpenTok session.
```

Comment:
Please copy (with any required edits) the docs for the REST method: https://tokbox.com/developer/rest/#starting_experience_composer (for this description and the docs for the options). For example:

```suggestion
  * Starts an Experience Composer for an OpenTok session. The Experience Composer
  * instance is represented as a Render object.
  * For more information, see the
  * <a href="https://tokbox.com/developer/guides/experience-composer/">Experience Composer developer guide</a>.
```